message_sender/models.py

```diff
@@ -39,8 +39,9 @@ class Outbound(models.Model):
     Delivered is set to true when ack received because delivery reports patchy
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    to_addr = models.CharField(null=False, blank=False, max_length=500,
+    to_addr = models.CharField(null=True, blank=True, max_length=500,
```

Comment:
As I understand null is for making it nullable in the database and blank is for form validation. 
"For both string-based and non-string-based fields, you will also need to set blank=True if you wish to permit empty values in forms, as the null parameter only affects database storage (see blank)."